frontend/src/views/App.tsx

```diff
@@ -5,23 +5,27 @@ import {
   Route,
   Switch
 } from "react-router-dom"
+import Navbar from "../components/header/navbar/Navbar"
+import SimpleContainer from "../components/simpleContainer/SimpleContainer"
 import { RouteBuilder } from "../routes"
+import Login from "./Login/Login"
+import Main from "./Main/Main"
 
 const App = (): JSX.Element => {
   return (
     <Router>
-      <Switch>
-        <Route
-          path={RouteBuilder.toLogin()}
-          render={() => <React.Fragment />}
-        />
-        <Route
-          path={RouteBuilder.toRegister()}
-          render={() => <React.Fragment />}
-        />
-        <Route path={RouteBuilder.toMain()} render={() => <React.Fragment />} />
-        <Redirect exact from="/" to={RouteBuilder.toMain()} />
-      </Switch>
+      <Navbar />
+      <SimpleContainer>
+        <Switch>
+          <Route path={RouteBuilder.toLogin()} render={() => <Login />} />
+          <Route
+            path={RouteBuilder.toRegister()}
+            render={() => <React.Fragment />}
+          />
+          <Route path={RouteBuilder.toMain()} render={() => <Main />} />
```

Comment:
```suggestion
          <Route path={RouteBuilder.toMain()} render={Main} />
```
Tak też powinno zadziałać

Reply:
Wyrzuca błędy

Reply:
wyjaśniłam sprawę niżej